### PR TITLE
JavaScript capstone project

### PR DESCRIPTION
**Pull request by memelopez**
### This PR implements: 🆕 💻 🔥 🆒 💯 
- Use of JavaScript to make the website dynamic and build single page app. ✔️ 
- Use of ES6 syntax. ✔️ 
- Use of ES6 modules. ✔️ 
- Use of callbacks and promises. ✔️ 
- Use of webpack. ✔️ 
- Use of AAA pattern for testing. ✔️ 
- Use of Jest for testing. ✔️ 
- Correct use of Gitflow. ✔️ 
- Sends and receives data from APIs. ✔️ 
- JavaScript asynchronous code. ✔️ 
- Code reviews between team members. ✔️ 
- A home page showing a list of POKEMONS 🐯 that can be "liked". ✔️ 
- A popup window with more data about the POKEMON 🦁 that shows and creates comments. ✔️ 
- A well documented README file that has a link to a video with an overview of project. ✔️ 
- No linter errors ✅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team for guidance
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (FrederickMih) [inline README.md:98]: - [ ] Great README!. 
Please kindly fix this link. This will render our README professional:smiley:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @memelopez @Hadi111jafari  , 

A fantastic project so far!
There are some little issues that you still need to work on to prepare your project for the final evaluation but you are almost there! 





# To Highlights

-  Git/GitHub workflow has been followed :heavy_check_mark: 
-  Linters are correctly set up :heavy_check_mark: 
-  Pull Request has a title and summary :heavy_check_mark: 
-  Home page and Comment popup works as expected :heavy_check_mark: 

## Suggested changes
- [x] **Video Presentation**: Hi team, your video presentation is really interesting. Please let's also highlight a piece of the code we find interesting or proud of making as the requirement says
- [x] From your video presentation, it shows that the counter functions were get covered with unit tests but when I run the command `npm run test`, I get this console error.


Screenshot from 2022-04-23 11-30-04.

 Any idea?

If I'm not missing out on something, please let's make sure that these functions are covered with unit tests using the **Jest Framework**





Thanks :pray: 



_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Hadi111jafari) [reply README.md]: Functions are covered with unit tests are in testing brunch, it is not merged.
You can check that.
We tried as much as we could explain the code, but there was time shortage that the video should not be longer than 5 minutes.
And the link for MIT license is updated.
@FrederickMih

Comment (memelopez) [reply README.md]: #### Hi reviewers! 👋 

Thank you for your valuable feedback. We have made changes as requested:
- we have re-recorded the video to say what we are proud of in the code.
- we have fixed the link about the MIT license 
- and about the testing: we preferred not to merge the testing to our development branch, so it is a separate branch called testing. I hope this is not too inconvenient for you guys. 

Thanks again for your reviews and have a nice day! ✌🏼😘

**Review by mmsesay — APPROVED** [review]
Hi @memelopez & @Hadi111jafari, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md (hunk 1 above)
Comment (FrederickMih) [reply README.md]: > Functions are covered with unit tests are in testing brunch, it is not merged. You can check that. We tried as much as we could explain the code, but there was time shortage that the video should not be longer than 5 minutes. And the link for MIT license is updated. @FrederickMih

@memelopez @Hadi111jafari , great job for recording another video

**Pull request merged**